### PR TITLE
[TRANSLATION][BROWSEUI] Improve French (fr-FR) translation

### PR DESCRIPTION
**Pull request by fulup29**
JIRA issue: [CORE-18914](https://jira.reactos.org/browse/CORE-18914)

Before
go

After
move

### CODE REVIEW

**Review by KRosUser2 — APPROVED** [review]
Lgtm

**Comment by KRosUser2** [comment]
Please do a single commit for this

**Comment by HBelusca** [comment]
> Please do a single commit for this

We can squash&merge into one single commit with the github interface.

File: dll/win32/browseui/lang/fr-FR.rc [hunk 1]
```diff
@@ -334,7 +334,7 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_STANDARD_TOOLBAR, "|Retour|Avancer|Dossier parent|Recherche|Dossiers|Aller vers|Copier dans|Supprimer|Annuler|Affichage|Stop|Rafraîchir|Home|Map Drive|Déconnecter|Favoris|Historique|Plein écran|Propriétés|Couper|Copier|Coller|Options des dossiers||"
+    IDS_STANDARD_TOOLBAR, "|Retour|Avancer|Dossier parent|Recherche|Dossiers|Déplacer vers|Copier dans|Supprimer|Annuler|Affichage|Stop|Rafraîchir|Home|Map Drive|Déconnecter|Favoris|Historique|Plein écran|Propriétés|Couper|Copier|Coller|Options des dossiers||"
```
Comment (HBelusca) [inline dll/win32/browseui/lang/fr-FR.rc:337]: By the way, "Home" and "Map Drive" can be translated :)

Comment (KRosUser2) [reply dll/win32/browseui/lang/fr-FR.rc:337]: I'd suggest 
"Home" : Accueil
"Map Drive" : Connecter

**Comment by fulup29** [comment]
I'm gonna close my PR to resolve my issue with my repository, it will avoid the 10 commits.

**Pull request closed without merging**